app/webpacker/styles/application.scss

```diff
@@ -12,3 +12,4 @@ $govuk-image-url-function: frontend-image-url;
 @import "~govuk-frontend/govuk/all";
 
 @import "thisapp/pagination.scss";
+@import "thisapp/overrides.scss";
```

Comment:
Why the `thisapp` namespace?

Reply:
No particular reason other than it makes it very explicit that these are from (and for) specifically this application, rather than anywhere else in the import path. Happy to change or remove if it's a problem?

Reply:
I think that if the files are in the repo it's a given.